src/components/themed/AddressTile.js

```diff
@@ -109,11 +109,7 @@ class AddressTileComponent extends React.PureComponent<Props, State> {
 
       // Check is PaymentProtocolUri
       if (!!parsedUri.paymentProtocolURL && !parsedUri.publicAddress) {
-        const guiMakeSpendInfo: ?GuiMakeSpendInfo = await paymentProtocolUriReceived(parsedUri, coreWallet)
-
-        if (guiMakeSpendInfo) {
-          onChangeAddress(guiMakeSpendInfo)
-        }
+        await launchBitPay(parsedUri.paymentProtocolURL, { wallet: coreWallet }).catch(showError)
```

Comment:
This change appears to break payment protocol v1 support if you're pasting (or god forbid) typing in the URI. We still support v1 in the scan functionality so it should still be supported here as well.